data/variants.txt

```diff
@@ -459,6 +459,7 @@ ship "Carrier" "Carrier (Mark II)"
 		"A860 Atomic Thruster"
 		"A525 Atomic Steering"
 		"Hyperdrive"
+		"Electron Rifle" 123
```

Comment:
Given that these are the *sole* Navy variants with Mark II technology (with the unique exception of the jump ships, which are only seen on special occasions anyway), and given that the Navy likes to be efficient, I don't think it makes sense for them not to max out their rifles. At the very least, I would not put it on an non-rounded number like this.
Consider the other armed designs as well: the Heavy Cruiser has 20 rifles and 20 grenades, and the Alpha carrier (and we know the Alphas to be *exceptionally* efficient) has 120 rifles.

Reply:
Noted

Reply:
So 120 rifles on carriers, 20 rifles on cruisers, and 16 rifles on frigates?